File: ts/src/test/Exchange/test.fetchMarginMode.ts
import { Exchange } from "../../../ccxt";
import testMarginMode from './base/test.marginMode.js';

async function testFetchMarginMode (exchange: Exchange, skippedProperties: object, symbol: string) {
    const method = 'fetchMarginMode';
    const marginMode = await exchange.fetchMarginMode (symbol);
    testMarginMode (exchange, skippedProperties, method, marginMode);
    return true;
}

export default testFetchMarginMode;

File: julia/Ccxt/test/exchange/test.fetchMarginMode.jl
using Test
using Ccxt
function testFetchMarginMode(exchange, skippedProperties, symbol)

    method = "fetchMarginMode";
    marginMode = Base.fetch(fetchMarginMode(exchange, symbol));
    testMarginMode(exchange, skippedProperties, method, marginMode);
    return true
end
